src/init/features/apphosting/index.ts

```diff
@@ -62,7 +66,30 @@ export async function doSetup(
     }
   }
 
-  logBullet("First we need a few details to create your backend.");
+  logBullet("First we need a few details to create your backend.\n");
+
+  let webAppId: string;
+
+  const existingUserProjectWebApps = new Map(
+    (await listFirebaseApps(projectId, AppPlatform.WEB)).map((obj) => [
```

Comment:
this lists up to a max of 100 apps, do we need to consider the case of a user having > 100 apps in a project?